R/build-home-authors.R

```diff
@@ -56,6 +56,12 @@ data_home_sidebar_authors <- function(pkg = ".") {
     authors <- c(authors, "<a href='authors.html'>All authors...</li>")
   }
 
+  if (!is.null(pkg$citation)) {
```

Comment:
I think I'd prefer a separate `has_citation(path)` function in with `has_vignettes()` and `has_news()`